content_api/app/settings.py

```diff
@@ -47,6 +47,7 @@ def env(variable, fallback_value=None):
 OAUTH2_TOKEN_URL = '/token'
 OAUTH2_SCOPES = ['content_api']
 BCRYPT_GENSALT_WORK_FACTOR = 12
+PUBLIC_RESOURCES = ['publish']
```

Comment:
this could be set on the resource itself via [public_method](http://python-eve.org/authentication.html#custom-endpoint-security) . and shouldn't be there `assets` too?